### PR TITLE
Dev/1.0.0

### PR DESCRIPTION
**Pull request by alexjhawk**

### CODE REVIEW

**Comment by alexjhawk** [comment]
As a note, no functional changes were made in the 'Add _____ package' commits. The only modifications in those commits are to update the '@since' Javadoc tags and package paths. Functional changes are in the 'Add description getter to TagInfo' commit or newer.

**Comment by alexjhawk** [comment]
Additionally, the added GitHub actions have been tested and worked properly, including a test release of this library to Maven.

File: src/main/java/com/hms_networks/americas/sc/extensions/ExtensionsMain.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package com.hms_networks.americas.sc.extensions;
```
Comment (TomKimsey) [inline src/main/java/com/hms_networks/americas/sc/extensions/ExtensionsMain.java:1]: Whats the reason for including a class with a main method?

Comment (alexjhawk) [reply src/main/java/com/hms_networks/americas/sc/extensions/ExtensionsMain.java:1]: This main class/main method have no functionality. The Jars built by Maven include additional metadata like project version and name, but also the ability to double-click to launch the .jar using its main method. For consistency, and to not specify a main class which does not exist, I created this basic one.

**Comment by alexjhawk** [comment]
Pushed GH action changes to match https://github.com/hms-networks/sc-java-maven-starter-project/pull/2

**Review by it-hms — APPROVED** [review]

File: src/main/java/com/hms_networks/americas/sc/extensions/ExtensionsMain.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * Main class for the project. This class does not perform any functions, as it is contained in a
+ * library.
+ *
+ * @since 1.0.0
+ * @author HMS Networks, MU Americas Solution Center
+ */
+public class ExtensionsMain {
+
+  /**
+   * Main method for project main class.
```
Comment (oliver-walker-hms) [inline src/main/java/com/hms_networks/americas/sc/extensions/ExtensionsMain.java:13]:
```suggestion
   * Main method for the project main class.
```

**Review by oliver-walker-hms — CHANGES_REQUESTED** [review]

File: src/main/java/com/hms_networks/americas/sc/extensions/ExtensionsMain.java (hunk 2 above)
Comment (alexjhawk) [reply src/main/java/com/hms_networks/americas/sc/extensions/ExtensionsMain.java]: fixed

**Review by oliver-walker-hms — APPROVED** [review]

**Review by TomKimsey — APPROVED** [review]

**Pull request merged**